fastlane/.env.example

```diff
@@ -0,0 +1,4 @@
+# Add your SLACK_URL & ACCESS_TOKEN HERE
+
+SLACK_URL="https://hooks.slack.com/services/..."
+ACCESS_TOKEN = "xoxp-106043..."
```

Comment:
is this an actual access token? if so need to change/remove it.

Reply:
Hi Nathan,
No it is not. I just added it as an example.